KrispyTech/airneis
# AIRNEIS-59: Display categories on mobile

# DONE

- Fixed get categories and get category by ID endpoint
- Created category page for mobile 
![image](https://github.com/KrispyTech/airneis/assets/117679829/6c588058-cb27-4aa7-8a6a-e7f9ee86efc4)
![image](https://github.com/KrispyTech/airneis/assets/117679829/cec0683e-4c06-49fc-80de-bea70e855203)
- Listed categories on homepage so you can test with the category you want
![image](https://github.com/KrispyTech/airneis/assets/117679829/df65a54d-26c4-4e09-b7f8-5c4080453325)

# To test 
- Add the env variables from the .env.exemple into your .env file
- Have some categories and products to your DB.
- Launch the project
-  You should be on the home page
-  Go to a category page
